plasmapy/physics/parameters.py

```diff
@@ -34,22 +34,18 @@
 
 def _grab_charge(ion, z_mean=None):
     """Utility function to merge two possible inputs for particle charge.
-
```

Comment:
Do you think you could revert the whitespace changes here?

Reply:
yes of course, I will change it and add some tests on the next update